### PR TITLE
[unittest] Add verification of accuracy for training metrics

### PR DESCRIPTION
**Pull request by kparichay**
This patch add verification of accuracy for training metrics like loss and accuracy
in capi test cases

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: test/tizen_capi/unittest_tizen_capi.cpp [hunk 1]
```diff
@@ -24,6 +24,42 @@
...
+
+  EXPECT_FLOAT_EQ(std::atof(summary), train_loss);
+  free(summary);
+
+  status = ml_train_model_get_summary(
+    model, (ml_train_summary_type_e)ML_TRAIN_SUMMARY_MODEL_VALID_LOSS,
+    &summary);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  EXPECT_FLOAT_EQ(std::atof(summary), valid_loss);
```
Comment (zhoonit) [inline test/tizen_capi/unittest_tizen_capi.cpp:51]: if given argument is invalid, `std::atof` has undefined behavior. 
how about using `std::stod`?

**Review by zhoonit — APPROVED** [review]
Awesome :)

File: test/tizen_capi/unittest_tizen_capi.cpp (hunk 1 above)
Comment (kparichay) [reply test/tizen_capi/unittest_tizen_capi.cpp]: done.

File: nntrainer/include/neuralnet.h [hunk 2]
```diff
@@ -52,6 +52,16 @@ namespace nntrainer {
  */
 typedef enum { NET_KNN, NET_REG, NET_NEU, NET_UNKNOWN } NetType;
 
+/**
+ * @brief     Statistics from running or training a model
+ */
+typedef struct _run_stats {
+  float accuracy; /** accuracy of the model */
```
Comment (jijoongmoon) [inline nntrainer/include/neuralnet.h:59]: I thought we agreed not to use variable which start with "_".

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: nntrainer/include/neuralnet.h (hunk 2 above)
Comment (kparichay) [reply nntrainer/include/neuralnet.h:59]: Updated to `RunStats_`.

**Pull request merged**